components/game.tsx

```diff
@@ -28,7 +28,7 @@ export default function Game() {
         // Filter out questions which give away their answers
         .filter((item) => !item.label.includes(String(item.year)))
         .filter((item) => !item.description.includes(String(item.year)))
-        .filter((item) => !item.description.includes(String("st century" || "nd century" || "th century")))
+        .filter((item) => !/(?:th|st|nd)[ -]century/.test(item.description))
```

Comment:
Let's make this case-insensitive too 

```suggestion
        .filter((item) => !/(?:th|st|nd)[ -]century/i.test(item.description))
```

Reply:
Done!